portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A class that represents a comment. */
+public final class Comment {
+
```

Comment:
remove extra new line

Reply:
Sure. They had it in the example so I thought it was there for a reason lol.

Reply:
I realize that in the examples they have an empty line there in every java class. And it does look better, don't you think?

Reply:
It's more about convention. The convention is to not have it.

Reply:
Okay got it